### ISSUE TITLE
Curve and Profile are purely 2-D - adjust code and docs

### ISSUE THREAD

**Issue by antnieszka** [DataMedSci/beprof#114]
Add some additional test too.

**Comment by antnieszka** [comment]
My first look at Curve class:
```py
>>> a = [[1.1,1.2,1.3],[2.1,2.2,2.3],[3.1,3.2,3.3], [4.1,4.2,4.3], [5.1,5.2,5.3]]
>>> c = Curve(a)
>>> c
Curve([[ 1.1,  1.2,  1.3],
       [ 2.1,  2.2,  2.3],
       [ 3.1,  3.2,  3.3],
       [ 4.1,  4.2,  4.3],
       [ 5.1,  5.2,  5.3]])
>>> c.x
DataSet([ 1.1,  2.1,  3.1,  4.1,  5.1])
>>> c.y
DataSet([ 1.2,  2.2,  3.2,  4.2,  5.2])
>>> c.z
Traceback (most recent call last):
  File "<input>", line 1, in <module>
AttributeError: 'Curve' object has no attribute 'z'
```
This looks fine, but how can we get the third column (z). There is no
```py
    @property
    def z(self):
        return self[:, 2].view(DataSet)
```
so we can't get the below:
```py
>>> c.z
DataSet([ 1.3,  2.3,  3.3,  4.3,  5.3])
```

**Comment by grzanka** [comment]
I think we can now assume that the Curves and Profiles are purely 2-D and adjust the code, comments and documentation to such version.

**Comment by antnieszka** [comment]
Ok, I'll rename this issue and add it to milestone 0.1.1.

**Comment by antnieszka** [comment]
Closed in #123
